site/src/components/power-the-polls-form/power-the-polls-form.tsx

```diff
@@ -163,10 +166,13 @@ export class PowerThePollsForm {
                </label>
 
                <label>
-                  Mobile phone
+                  Mobile phone<span class="ak-required-flag">*</span>
```

Comment:
I think we're ditching the `ak-required-flag` class - @nexussays can confirm

Reply:
Yes, holdover from the old design

Reply:
Remove!